pom.xml

```diff
@@ -356,9 +355,51 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:30min pending tasks in fixing findbugs.
```

Comment:
@sebing These should be proper sentences, e.g. `Continue fixing FindBugs violations started in #517`, etc.